### PR TITLE
Build inbox

### PR DESCRIPTION
**Pull request by fetticcine**
preview of what this would like

screen shot 2018-07-15 at 8 44 17 pm

### CODE REVIEW

File: src/components/UserRow.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Text, View, Image } from 'react-native';
+import PropTypes from 'prop-types';
+import CardSection from './common/CardSection';
+import colors from '../assets/colors';
+
+const UserRow = ({ image, senderName, msgPreviewText, timeStamp }) => {
```
Comment (AbdulAli19) [inline src/components/UserRow.js]: where else would this be used? why is the name so generic?

File: src/components/common/CardSection.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { View } from 'react-native';
+
+const CardSection = props => (
```
Comment (AbdulAli19) [inline src/components/common/CardSection.js]: do we have plans to use this component beyond where it's used currently? if not, it seems a little too unnecessary of an abstraction

File: src/components/UserRow.js (hunk 1 above)
Comment (AbdulAli19) [reply src/components/UserRow.js]: change this to `MessageRow` and create a Message folder in `/components`

Comment (fetticcine) [reply src/components/UserRow.js]: @AbdulAli19 - good point. Changed name to 'MessageRow'. 

Also created a `Message folder in /components to house our message components.

File: src/components/common/CardSection.js (hunk 2 above)
Comment (fetticcine) [reply src/components/common/CardSection.js]: @AbdulAli19 - agreed. I've added the styling from this component directly to "MessageRow" and will delete this "CardSection" component.

File: src/screens/Inbox.js [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
-export default class Inbox extends React.Component {
+import MessageListContainer from '../components/message/MessageListContainer';
+
+class Inbox extends React.Component {
   render() {
     return (
-      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
-        <Text>Inbox Screen</Text>
-      </View>
+      <MessageListContainer />
```
Comment (fetticcine) [inline src/screens/Inbox.js]: this isn't going to look like this when we add tabs (messages + notifications) so let's revert this to the placeholder text for now. it's cool to do this for testing purposes but what we end up pushing should ideally only move us towards the final product vision

File: src/components/message/MessagesList.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+import MessageRow from '../message/MessageRow';
+
+
+const imageStyle = {
+    height: 40,
+    width: 40,
+    borderRadius: 20,
+    marginRight: 10
+};
```
Comment (fetticcine) [inline src/components/message/MessagesList.js:13]: can we add TODO's here to remind us to make these pixel sizes dynamic (aka based on screen size and not fixed)

Comment (fetticcine) [reply src/components/message/MessagesList.js:13]: https://facebook.github.io/react-native/docs/dimensions.html

File: src/components/message/MessageRow.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  iconNameContainerStyle: {
+    justifyContent: 'space-between',
+    flexDirection: 'row',
+  },
+  iconStyle: {
+    height: 40,
+    width: 40,
+    borderRadius: 20,
+    marginRight: 10,
+  },
```
Comment (fetticcine) [inline src/components/message/MessageRow.js:57]: same here with the dynamic styles

File: App.js [hunk 6]
```diff
@@ -1,5 +1,11 @@
 import React from 'react';
+import { View } from 'react-native';
+
+import MessageListContainer from './src/components/message/MessageListContainer';
+import BaseButton from './src/components/common/BaseButton';
+import colors from './src/assets/colors';
 import AppNavigator from './src/navigator';
+
```
Comment (fetticcine) [inline App.js]: yo get rid of these imports, this file should stay the same for now at least

**Pull request merged**